web/libs/editor/src/stores/RegionStore.js

```diff
@@ -280,6 +280,15 @@ export default types
           }
         }
 
+        // Handle video rectangle regions - they have sequence with frames
+        if (region.type === "videorectangleregion" && region.sequence && region.sequence.length > 0) {
+          // Get the first keyframe's frame number
+          const firstKeyframe = region.sequence[0];
+          if (firstKeyframe && typeof firstKeyframe.frame === "number") {
+            return firstKeyframe.frame;
+          }
+        }
```

Comment:
I'd simplify code, now it's overcautious
```suggestion
        // Handle video rectangle regions - they have sequence with frames
        if (region.type === "videorectangleregion" && region.sequence?.[0]) {
          // Return the first keyframe's frame number
          return region.sequence[0].frame;
        }
```